lib/gooddata/lcm/actions/synchronize_ldm.rb

```diff
@@ -74,14 +77,15 @@ def sync_segment_ldm(params, segment_info)
           from = params.development_client.projects(from_pid) || fail("Invalid 'from' project specified - '#{from_pid}'")
 
           GoodData.logger.info "Creating Blueprint, project: '#{from.title}', PID: #{from_pid}"
-          blueprint = from.blueprint(include_ca: params.include_computed_attributes.to_b)
+          blueprint = from.blueprint(include_ca: params.include_computed_attributes.to_b, include_deprecated: (params[:include_deprecated] || true))
           maql_diff = nil
           previous_master = segment_info[:previous_master]
           diff_against_master = %w(diff_against_master_with_fallback diff_against_master)
             .include?(params[:synchronize_ldm].downcase)
           if previous_master && diff_against_master
-            maql_diff_params = [:includeGrain]
-            maql_diff_params << :excludeFactRule if exclude_fact_rule
+            maql_diff_params = { :includeGrain => true }
+            maql_diff_params[:excludeFactRule] = true if exclude_fact_rule
+            maql_diff_params[:includeDeprecated] = params[:include_deprecated] || false
             maql_diff = previous_master.maql_diff(blueprint: blueprint, params: maql_diff_params)
```

Comment:
- why changing includeGrain and excludeFactRule?
- includeDeprecated should same as excludeFactRule which will add param includeDeprecated to URL only if includeDeprecated=true: 
maql_diff_params << : includeDeprecated if includeDeprecated

Reply:
I fixed as your comment